src/Security/VoterV2/BeneficiaryVoter.php

```diff
@@ -21,8 +23,8 @@ public function __construct(private readonly UserHelper $helper)
      */
     protected function supports(string $attribute, $subject): bool
     {
-        return self::UPDATE === $attribute
-            && $subject instanceof Beneficiaire;
+        return (self::MANAGE === $attribute && !$subject)
+            || (self::UPDATE === $attribute && $subject instanceof Beneficiaire);
```

Comment:
Too many conditions in one line, you can make it clearer using `if(..) return` or moving logic parts to differents methods with explicits names. This could be discussed as a good practice